backend/scripts/seed_data.py

```diff
@@ -165,51 +169,91 @@
         test_results=[
             C3TestResult(
                 id=1,
-                name="test1",
-                status=C3TestResultStatus.FAIL,
-                category="",
-                comment="",
-                io_log="",
-            ),
-            C3TestResult(
-                id=2,
-                name="test2",
+                name="docker/compose-and-basic_armhf",
                 status=C3TestResultStatus.PASS,
-                category="",
+                category="Docker containers",
                 comment="",
-                io_log="",
+                io_log=dedent(
+                    """
+                    test Pulling 
+                    test Pulled 
+                    Network 4506_default  Creating
+                    Network 4506_default  Created
+                    Container 4506-test-1  Creating
+                    Container 4506-test-1  Created
+                    Container 4506-test-1  Starting
+                    Container 4506-test-1  Started
+                    Container 4506-test-1  Stopping
+                    Container 4506-test-1  Stopped
+                    Container 4506-test-3  Creating
+                    Container 4506-test-2  Creating
+                    Container 4506-test-1  Recreate
+                    Container 4506-test-3  Created
+                    Container 4506-test-2  Created
+                    Container 4506-test-1  Recreated
+                    Container 4506-test-1  Starting
+                    Container 4506-test-1  Started
+                    Container 4506-test-2  Starting
+                    Container 4506-test-2  Started
+                    Container 4506-test-3  Starting
+                    Container 4506-test-3  Started
+                    3
```

Comment:
I'm wondering if it's fine to have this random number in this list

Reply:
This is just a string that isn't parsed so it doesn't break anything. But I do agree with you this random number is weird. So I went back to the [source](https://certification.canonical.com/hardware/202011-28409/submission/345403/test/90212/result/37922493/) to make sure I copied it correctly. And sure enough the 3 exists :sweat_smile: